addons/crm/tests/test_crm_lead.py

```diff
@@ -990,6 +990,61 @@ def test_phone_mobile_update(self):
         self.assertEqual(lead.phone, self.test_phone_data[1])
         self.assertFalse(lead.phone_sanitized)
 
+    @users('user_sales_manager')
+    def test_incoming_email_automatic_lead_assignment(self):
+        # create two users each belonging to one of the teams
```

Comment:
Can't you reuse existing data defined in sales_team and here ? Or at least use existing data and define additional one, but not redefine everything ?